docs/deploy/README.md

```diff
@@ -0,0 +1,7 @@
+# Deploy
```

Comment:
I think we can put all the contents here. Or what else do you expect to put under "deploy"?

Also, in the kubernetes docs it's called "Setup"